.github/release-please.yml

```diff
@@ -11,3 +11,7 @@ branches:
     manifest: true
     handleGHRelease: true
     branch: 26.1.x
+  - releaseType: java-yoshi
+    manifest: true
+    handleGHRelease: true
+    branch: protobuf-4.x-rc
```

Comment:
Curious why this one does not have `bumpMinorPreMajor: true`? What exactly does it do?

Reply:
It makes release-please to do a minor bump if breaking changes are introduced, as long as the current released version is pre-major (e.g. `0.1.0`). Libraries bom is on its 26th Major release so this has no effect.

https://github.com/googleapis/release-please/blob/009c2a61a038843957e7002aa1c32e317b654aa1/docs/cli.md?plain=1#L105